internal/provider/provider.go

```diff
@@ -62,6 +63,7 @@ func configure(client *basistheory.Client, provider *schema.Provider) func(conte
 			return map[string]interface{}{
 				"client": client,
 				"api_key":  data.Get("api_key"),
+				"api_url":  data.Get("api_url"),
```

Comment:
this doesn't look like it's used anywhere

Reply:
The `api_url` is needed to call the raw HTTP API in delete apple pay domains. This `if` branch is executed in the tests since tests pass in a custom client.